controllers/v1/project/templates.js

```diff
@@ -666,12 +667,16 @@ module.exports = class ProjectTemplates extends Abstract {
 
        async details(req) {
         return new Promise(async (resolve, reject) => {
+            
             try {
-
+                (req.params._id) ? req.params._id=req.params._id : req.params._id="";
+                (req.query.link) ? req.query.link=req.query.link : req.query.link="";
+                let userId="";
+                //req.userDetails.userInformation.userId
                 let projectTemplatesDetails = 
                 await projectTemplatesHelper.details(
-                    req.params._id,
-                    req.userDetails.userInformation.userId
```

Comment:
@VISHNUDAS-tunerlabs dont remove "req.userDetails.userInformation.userId" it will impact logged in user flow